modules/local_storage/commerce_stock_local.install

```diff
@@ -292,4 +266,12 @@ function commerce_stock_local_install() {
       'parent_id' => 1,
     ])
     ->execute();
+
```

Comment:
Creates a main stock location entity, so the users don't need to install one, if the only need one.